src-java/base-topology/base-storm-topology/src/main/java/org/openkilda/wfm/share/service/SpeakerFlowSegmentRequestBuilder.java

```diff
@@ -277,11 +277,11 @@ private FlowSegmentRequestFactory makeIngressSegmentRequest(
         return IngressFlowSegmentRequestFactory.builder()
                 .messageContext(messageContext)
                 .metadata(makeMetadata(path, ensureEqualMultiTableFlag(
-                        flowSide.isMultiTableSegment(), segmentSide.isMultiTable(),
-                        String.format("First flow(id:%s, path:%s) segment and flow level multi-table flag values are "
-                                              + "incompatible to each other - flow(%s) != segment(%s)",
+                        path.isSrcWithMultiTable(), segmentSide.isMultiTable(),
```

Comment:
Why direct request via path object? As for me, request via flowSide is more flexible.

Reply:
Because I removed multitable-related info from adapter level. After this changes flow may have multiple paths with different multitable mode on the same endpoint so from flow level we can't get this info in a simple way.